notes/stories/film/film-scifi-1990s.st.txt

```diff
@@ -10571,6 +10571,46 @@ father and son [we saw brief flashbacks to Truman with his dad who went missing
 mother and son [we saw Truman with his mom from time to time]
 
 
+movie: The XFiles Fight the Future (1998)
+=========================================
+
+:: Title
+The X-Files: Fight the Future
+
+:: Description
+The X-Files (also known as The X-Files: Fight the Future) is a 1998 American
+science fiction thriller film based on Chris Carter's television series of
+the same name, which revolves around fictional unsolved cases called the
+X-Files and the characters solving them.
+
+:: Date
+1998-06-19
+
+:: References
+https://en.wikipedia.org/wiki/The_X-Files_(film)
+
+:: Ratings
+3 <paul>
+
+:: Genre
+sci-fi
+
+:: Major Themes
+secret society conspiracy theory [The story follows the FBI agents Mulder and Skully as they attempt to get to the bottom of a global conspiracy being perpetrated by an unnamed shadowy organization to prepare the way for Earth to be colonized by space aliens.]
+subversive life form from outer space [A shadowy organization with a global presence was preparing the way for space aliens to colonize Earth.]
+extraterrestrial being [Powerful space aliens were using a shadowy organization to spread a virus around the world as a prelude to colonization.]
+human characters in contrast [The intuition-trusting, paranormal-believing Agent Mulder was pointedly juxtaposed with his rational-minded, skeptical partner Agent Skully.]
+human interpersonal relationship [Much focus was placed on the relationship between the buddy FBI agents Mulder and Skully. In one subplot, Mulder was urging Skully to stay on the force when she wanted to quit the FBI and return to practicing medicine.]
+
+:: Minor Themes
+terrorism [A shadowy group staged a terrorist bombing of a Dallas building to cover up how a young boy and a fireman had really died.]
+pedophilia [Efforts were made to discredit Dr. Kurtzweil by pinning the selling of naked pictures of children on him.]
+flying saucer [An enormous saucer shaped vessel of alien origin emerged from the Antarctic ice and flew off into the sky.]
+
+:: Other Keywords
+black helicopter conspiracy theory [Menacing black helicopters pursued Mulder and Skully through a cornfield.]
```

Comment:
Huh?

Reply:
You didn't know? https://en.wikipedia.org/wiki/Black_helicopter

Reply:
mm okay

Reply:
I was surprised to see a conspiracy theory in other keywords rather than as a theme, since we have a general theme for them. I suppose it was too interpretive? But if so I'm not sure it's worth having in keywords.

Reply:
Too interpretative was exactly my thinking. Mulder and Skully were buzzed by some unmarked black helicopters. I mean knowing the context of the show you can guess what the writers are getting at - but I just left it for future generations to sort out.